phoenix-pherf/src/main/java/org/apache/phoenix/pherf/jmx/MonitorManager.java

```diff
@@ -135,10 +154,7 @@ public synchronized boolean shouldStop() {
         return shouldStop;
     }
 
-    public synchronized void stop() {
-        this.shouldStop = true;
-    }
-
+    @SuppressWarnings("unused")
```

Comment:
I found it useful for testing and debugging. I just didn't leave any of the calling code in. I didn't want to have to re-add it later. I can remove it you don't like it though.